components/de_amp/common/features.cc

```diff
@@ -13,14 +13,7 @@ namespace features {
 
 // When enabled, Brave will try to de-AMP a page i.e. load the canonical,
 // non-AMP version if the page is an AMP page.
-const base::Feature kBraveDeAMP {
-  "BraveDeAMP",
-#if BUILDFLAG(IS_ANDROID)
-      base::FEATURE_DISABLED_BY_DEFAULT
-#else
-      base::FEATURE_ENABLED_BY_DEFAULT
-#endif
-};
+const base::Feature kBraveDeAMP{"BraveDeAMP", base::FEATURE_ENABLED_BY_DEFAULT};
```

Comment:
do we want it to be enabled by default? An original issue says that it will be controlled by Griffin.

Reply:
I think the idea was to have it enabled by default, and there's a feature flag that we could use to turn it off via Griffin in case there are issues. Can you confirm @pes10k

Reply:
Yep, exactly right @ShivanKaul